OpenShift/helm/nexus-repository-manager/Chart.yaml

```diff
@@ -1,13 +1,23 @@
-name: nexus-repository-manager
-version: 0.1
-description: A reference OpenShift deployment for Nexus Repository Manager
+apiVersion: v1
+name: sonatype-nexus
+version: 0.1.0
+appVersion: 3.20.1-01
+description: Sonatype Nexus is an open source repository manager
 keywords:
-- quickstart
-  ci
-  repository-manager
-  nexus3
-  nexus
-  sonatype
-home: https://www.sonatype.com/
+  - artifacts
+  - dependency
+  - management
+  - sonatype
+  - nexus
+  - repository
```

Comment:
should we retain any of the previous keywords?

Reply:
sure, I'll update